v2/reader.go

```diff
@@ -34,6 +51,10 @@ func NewReader(r io.ReaderAt) (*Reader, error) {
 	return cr, nil
 }
 
+func (r *Reader) CARv1Size() uint64 {
```

Comment:
This is nice to have for the storage market.

Reply:
Perhaps there's something I'm missing, but why is `r.Carv1Size()` easier to do than `r.Header.Carv1Size`? You barely save any keystrokes, and now we have an extra bit of API to document and learn to use :) I'd rather only add more ways of accomplishing the same if they allow actually saving a significant amount of code or complexity, but this doesn't feel like it to me.

Reply:
@mvdan Didn't realize this was exported :p My bad.